### DISCUSSION TITLE
Error with the package (Edited fixed after the update)

### DISCUSSION THREAD

**Discussion by krenarxh** [tasteui/tasteui#568] [Q&A] [answered]
Hi I am having an error on my project, not understanding it.

**Attribute "TallStackUi\Foundation\Attributes\SkipDebug" cannot target parameter (allowed targets: property)**

It was working really well, when I pulled it from the git repository it gave me this error.

![Screenshot from 2024-07-24 10-13-25](https://github.com/user-attachments/assets/96a6a737-6945-495c-af57-2a91e9f5546f)

**Comment by devajmeireles** [comment, marked as answer]
We already released a new version to solve this issue that only happens starting from Laravel 11.17
